bridge/client/app/_components/ktb-notification-bar/ktb-notification-bar.component.spec.ts

```diff
@@ -30,7 +30,7 @@ describe('KtbNotificationBarComponent', () => {
   });
 
   it('should add and remove notifications', () => {
-    let notifications = fixture.debugElement.queryAll(By.css('.page-note'));
+    let notifications = fixture.debugElement.queryAll(By.css('ktb-notification'));
```

Comment:
Any chance that these tests go into UI tests to avoid accessing the component? Unit test can look differently, inject the notification service and check if notifications set, the observable contains the number of elements or so? Also for hide, if notification service was called and contains proper number?

Reply:
I can only UI test the success message so far.
`info`: We disabled the version check for Cypress
`error`: Weird behavior. If an HTTP error is thrown, it is logged two times in the console (don't know where the root cause is so far) and this lets Cypress fail the test. Disabling the `uncaught:exception` event does not have any effect